WordPress/src/main/java/org/wordpress/android/ui/AppLogViewerActivity.java

```diff
@@ -140,14 +141,8 @@ private void copyAppLogToClipboard() {
     @Override
     public boolean onCreateOptionsMenu(Menu menu) {
         super.onCreateOptionsMenu(menu);
-        // Copy to clipboard button
-        MenuItem item = menu.add(Menu.NONE, ID_COPY_TO_CLIPBOARD, Menu.NONE, R.string.copy_text);
-        item.setShowAsAction(MenuItem.SHOW_AS_ACTION_IF_ROOM);
-        item.setIcon(R.drawable.ic_copy_white_24dp);
-        // Share button
-        item = menu.add(Menu.NONE, ID_SHARE, Menu.NONE, R.string.reader_btn_share);
-        item.setShowAsAction(MenuItem.SHOW_AS_ACTION_IF_ROOM);
-        item.setIcon(R.drawable.ic_share_white_24dp);
```

Comment:
Since both the `ID_SHARE` and `ID_COPY_TO_CLIPBOARD` constants are not being used anymore, I think it's safe to remove them from this class?

Reply:
Good catch! Removed them 👍